Core/src/IfcGeometryConverter/CurveConverter.h

```diff
@@ -1159,52 +1407,79 @@ namespace OpenInfraPlatform {
 				{
 					//	ABSTRACT SUPERTYPE OF IfcIntersectionCurve, IfcSeamCurve																//
 
-					throw oip::UnhandledException(surface_curve);
+					throw oip::UnhandledException(surfaceCurve);
 
 					/*
-					// (1/2) IfcIntersectionCurve SUBTYPE OF IfcSurfaceCurve
-					std::shared_ptr<typename IfcEntityTypesT::IfcIntersectionCurve> intersection_curve =
+					// IfcIntersectionCurve SUBTYPE OF IfcSurfaceCurve
+					std::shared_ptr<typename IfcEntityTypesT::IfcIntersectionCurve> intersectionCurve =
 						std::dynamic_pointer_cast<typename IfcEntityTypesT::IfcIntersectionCurve>(ifcCurve);
-					if (intersection_curve)
+					if (intersectionCurve)
 					{
 						// TO DO: implement
 					}
 
-					// (2/2) IfcSeamCurve SUBTYPE OF IfcSurfaceCurve
-					std::shared_ptr<typename IfcEntityTypesT::IfcSeamCurve> seam_curve =
+					// IfcSeamCurve SUBTYPE OF IfcSurfaceCurve
+					std::shared_ptr<typename IfcEntityTypesT::IfcSeamCurve> seamCurve =
 						std::dynamic_pointer_cast<typename IfcEntityTypesT::IfcSeamCurve>(ifcCurve);
-					if (seam_curve)
+					if (seamCurve)
 					{
 						// TO DO: implement
 					}
 					*/
 				}
 
+#if defined(OIP_MODULE_EARLYBINDING_IFC4X3_RC2)
+				/*! \brief Converts \c IfcSegment and its subtypes to a series of points.
+				* \param[in] ifcloop				The \c IfcSegment to be converted.
+				* \param[out] loop					The series of points.
+				*/
+				void convertIfcSegment(const EXPRESSReference<typename IfcEntityTypesT::IfcSegment>& segment,
+					std::vector<carve::geom::vector<3>>& loop

```

Comment:
`std::vector<carve::geom::vector<3>>& segmentStartPoints` is missing as a parameter (similarly to other functions)

Reply:
Resolved: f641dfc